dspace-api/src/main/java/org/dspace/app/audit/AuditConsumer.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.audit;
+
+import java.util.List;
+import java.util.UUID;
+
+import org.dspace.app.audit.factory.AuditServiceFactory;
+import org.dspace.core.Context;
+import org.dspace.event.Consumer;
+import org.dspace.event.Event;
+import org.dspace.services.ConfigurationService;
+import org.dspace.services.factory.DSpaceServicesFactory;
+
+/**
+ * Class to store all received events in the audit system, if auditing is enabled.
+ *
+ * @author Andrea Bollini (andrea.bollini at 4science.it)
+ * @author Stefano Maffei (stefano.maffei at 4science.com)
+ */
+
+public class AuditConsumer implements Consumer {
+    private AuditService auditService;
+    private ConfigurationService configurationService;
+    private List<Integer> meaningfulEvents;
+
+
+    public void initialize() throws Exception {
+        auditService = AuditServiceFactory.getInstance().getAuditService();
+        configurationService = DSpaceServicesFactory.getInstance().getConfigurationService();
+        meaningfulEvents = List.of(Event.MODIFY_METADATA, Event.CREATE, Event.DELETE,
+            Event.REMOVE);
+    }
+
+    /**
+     * Consume a content event
+     * 
+     * @param ctx   DSpace context
+     * @param event Content event
+     */
+    public void consume(Context ctx, Event event) throws Exception {
```

Comment:
Please add `@Override` to this method as well as to `end()` and `finish()` below.  It's best to always add this annotation when you are overriding methods.